src/test_svg_to_png.py

```diff
@@ -0,0 +1,15 @@
+import sys
+
+from svg_to_png.svg_to_png import convert_svg_to_png
+
+def main():
+    if len(sys.argv) != 3:
```

Comment:
NIT: argparse is a cleaner way to do this these days

https://docs.python.org/3/library/argparse.html

Doesn't matter for our use case tho :)